src/cow-react/modules/wallet/web3-react/connection/coinbase.tsx

```diff
@@ -38,7 +38,7 @@ const [web3CoinbaseWallet, web3CoinbaseWalletHooks] = initializeConnector<AsyncC
             new m.CoinbaseWallet({
               actions,
               options: {
-                url: RPC_URLS[SupportedChainId.MAINNET],
+                url: getRpcUrls(SupportedChainId.MAINNET)[0],
```

Comment:
Why only the first item? Before it was getting the full array if I read this right

Reply:
Previously this was getting a single URL because RPC_URLS was only returning a single URL.

Coinbase expects a single URL unfortunately, and doesn't support an array of it as far as I can tell.

Reply:
ubnernit: just add a small comment saying sth  like

`// Set first RPC url from mainnet (coinbase is limited to use only one RPC)`